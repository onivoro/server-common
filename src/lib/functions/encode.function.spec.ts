import { decode } from './decode.function';
import { encode } from './encode.function';

describe('encode', () => {
    it('worx for hacky situations', () => {
        const encoded = encode({ key: 'value' });
        expect(decode(encoded).key).toMatchSnapshot();
    });

});