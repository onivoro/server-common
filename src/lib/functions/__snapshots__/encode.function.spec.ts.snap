// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`encode worx for hacky situations 1`] = `"value"`;
